src/stream/src/executor/top_n_appendonly.rs

```diff
@@ -84,7 +84,8 @@ pub struct InnerAppendOnlyTopNExecutor<S: StateStore> {
     managed_state: ManagedTopNState<S>,
 
     /// In-memory cache of top (N + N * `TOPN_CACHE_HIGH_CAPACITY_FACTOR`) rows
-    cache: TopNCache,
+    /// TODO: support WITH TIES
```

Comment:
May add some explicit failure (e.g. `unimplemented!()` ) somewhere

Reply:
The type is already explict I think. I will take care when implement in frontend 🤪